edit/gatekeeper-constraint.vue

```diff
@@ -64,6 +64,16 @@ export default {
   },
 
   data() {
+    this.value.spec.match = this.value.spec.match || {};
+    this.value.spec.match.labelSelector = this.value.spec.match.labelSelector || {};
+    this.value.spec.match.labelSelector.matchExpressions = this.value.spec.match.labelSelector.matchExpressions || [];
+
+    this.value.spec.match = this.value.spec.match || {};
+    this.value.spec.match.namespaceSelector = this.value.spec.match.namespaceSelector || {};
+    this.value.spec.match.namespaceSelector.matchExpressions = this.value.spec.match.namespaceSelector.matchExpressions || [];
+
+    this.value.spec.enforcementAction = this.value.spec.enforcementAction || ENFORCEMENT_ACTION_VALUES.DENY;
```

Comment:
This sort of thing normally goes in the model [applyDefaults](https://github.com/rancher/dashboard/blob/master/plugins/steve/resource-instance.js#L1037).